### PR TITLE
style(Tabs.js): introduce variables for more flexibility

### PR DESCRIPTION
**Pull request by ASchmidt1024**
style(Tabs.js): change margin-bottom property in .tab-navigation to use a custom CSS variable
style(Tabs.js): change border-top-left-radius and border-top-right-radius properties in .tab-navigation li to use a custom CSS variable
style(Tabs.js): change color property in .tab-navigation li to use a custom CSS variable
style(Tabs.js): change background-color property in .tab-navigation li::after to use a custom CSS variable
style(Tabs.js): change border-top-left-radius and border-top-right-radius properties in .tab-navigation li::after to use a custom CSS variable
style(Tabs.js): change background-color property in .tab-navigation li:hover to use a custom CSS variable
style(Tabs.js): change background-color property in .tab-navigation li.active to use a custom CSS variable
style(Tabs.js): change color property in .tab

### CODE REVIEW

**Comment by ASchmidt1024** [comment]
@Weedshaker Was meinst?

File: src/es/components/molecules/tabs/Tabs.js [hunk 1]
```diff
@@ -39,17 +39,17 @@ export default class Tabs extends Shadow() {
...
         }
         :host .tab-navigation {
-            border-bottom: 1px solid var(--m-gray-300);
+            border-bottom: var(--border-bottom, 1px solid var(--m-gray-300));
             list-style: none;
             display: flex;
             padding: 0;
             gap: 1.5em;
-            margin-bottom: 3em;
+            margin-bottom: var(--tab-navigation-margin-bottom, 3em);
```
Comment (Weedshaker) [inline src/es/components/molecules/tabs/Tabs.js:47]: try to use the same selectors within a css statement. ether use always --tab-navigation or avoid it:

--tab-navigation-margin-bottom vs --margin-bottom

File: src/es/components/molecules/tabs/Tabs.js [hunk 2]
```diff
@@ -58,26 +58,26 @@ export default class Tabs extends Shadow() {
             content: '';
             width: 100%;
             height: 3px;
-            background-color: var(--m-orange-300);
+            background-color: var(--tab-navigation-background-color, var(--color-disabled));
```
Comment (Weedshaker) [inline src/es/components/molecules/tabs/Tabs.js:61]: same here... --tab-navigation needed? if yes, use the same for all the elements within this css selector, which is going to make it easier to control in the future. or at least to guess the variable names.

**Review by Weedshaker — APPROVED** [review]

**Comment by ASchmidt1024** [comment]
@Weedshaker Thanks for your review. The prefix `--tab-navigation` is now history.

**Pull request merged**